app/helpers/attendance_helper.rb

```diff
@@ -6,4 +6,12 @@ def formatted_class_days
   def day_value
     @day ||= Time.zone.now.to_date
   end
+
+  def attendance_notice(student)
+    if student.attendance_score(current_course, current_course.number_of_days_since_start) >= 8
```

Comment:
student.attendance_score(current_course, current_course.number_of_days_since_start) >= 8 - move to model

remove AttendanceHelper